com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -304,8 +304,7 @@ public NetworkObject NetworkObject
                     m_NetworkObject = GetComponentInParent<NetworkObject>();
                 }
 
-                if (m_NetworkObject == null || NetworkManager.Singleton == null ||
-                    (NetworkManager.Singleton != null && !NetworkManager.Singleton.ShutdownInProgress))
+                if (m_NetworkObject == null && (NetworkManager.Singleton == null || !NetworkManager.Singleton.ShutdownInProgress))
```

Comment:
nit: While you're at it it may be nice to explain why we care about shutdown here

Reply:
That was a good idea and while I was in there I realized that the original LogLevel check was <= as opposed to just <.  I believe it got set because it was spamming when the default Normal LogLevel was set.